scripts/cat_suitability.py

```
@@ -0,0 +1,36 @@
+    return (d,tied_spec,tied_gen,max_spec)
+def parse(path):
+    out=[]
+    for ln in open(path):
+        ln=ln.strip()
+        if not ln: continue
+        p=ln.split(); out.append((p[0],p[1] if len(p)>1 else f'm{len(out)}'))
+    return out
+if __name__=='__main__':
+    print('name\td\ttied_spec\ttied_gen\tmax_spec')
+    for smi,name in parse(sys.argv[1]):
+        try: r=scan_one(smi)
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)